x/ethbridge/common/common.go

```diff
@@ -0,0 +1,12 @@
+package common

```

Comment:
I don't think this deserves its own package

Reply:
These functions/type are used across multiple sub-packages, so would have circular references if they were in the main parent package. Do you have a better suggestion?

Reply:
why don't you put it on `types`? or alternatively just use the validation function from geth directly to avoid circular deps

Reply:
nvm, working in types, will be in next push